File: CargoForYou/WebApp/ViewModels/ParcelViewModel.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApp.ViewModels
{
    public class ParcelViewModel
    {
        [JsonProperty(PropertyName = "id")]
        public int Id { get; set; }

        [JsonProperty(PropertyName = "weight")]
        public double Weight { get; set; }

        [JsonProperty(PropertyName = "width")]
        public double Width { get; set; }

        [JsonProperty(PropertyName = "height")]
        public double Height { get; set; }

        [JsonProperty(PropertyName = "depth")]
        public double Depth { get; set; }

        [JsonProperty(PropertyName = "price")]
        public double Price { get; set; }

        [JsonProperty(PropertyName = "fragile")]
        public bool Fragile { get; set; }

        [JsonProperty(PropertyName = "hazardous")]
        public bool Hazardous { get; set; }

        [JsonProperty(PropertyName = "perishable")]
        public bool Perishable { get; set; }
    }
}

File: Cargo4You/CargoWebApp/Scripts/ViewModels/ParcelViewModel.js
$("#Weight").on("change paste keyup", function () {
    calculatePrice();
});
$("#Width").on("change paste keyup", function () {
    calculatePrice();
});
$("#Height").on("change paste keyup", function () {
    calculatePrice();
});
$("#Depth").on("change paste keyup", function () {
    calculatePrice();
});
function calculatePrice() {
    $.get("/Parcels/CalculateParcelPrice", { weight: $("#Weight").val(), width: $("#Width").val(), depth: $("#Depth").val(), height: $("#Height").val() },
     function (returnedData){
         $("#Price").val(returnedData)+'€';
     })
}